lib/inets/doc/src/httpc.xml

```diff
@@ -291,6 +291,7 @@
         <v>http_options() = [http_option()]</v>
         <v>http_option() = {timeout,         timeout()} | 
                            {connect_timeout, timeout()} | 
+                           {upgrade_timeout, timeout()} |
```

Comment:
I wonder if there are any naming guidelines around this module. Should it be "upgrade_timeout" or "tls_upgrade_timeout"? it might not be easy to understand what the upgrade is.

Reply:
I've no strong opinion here, on the other hand we've `connect_timeout` and not `tcp_connect_timeout` (maybe TCP and TLS are implicitly assumed here).
Maybe @IngelaAndin could share opinion as well ?

Reply:
I've picked the proposal you've made earlier to have a dedicated timeout for upgrade for simplicity reasons.
In comparison to

1. re-using timeout
2. or making some timeout algebra (as suggested by Max in previous PR)

a dedicated timeout is easy to test (I can set only 2nd timeout in chain to zero) and also brings the biggest control to user.
Due to above I decided to start discussion with that proposal.

Timeout chain
=========
TCP TIMEOUT -> TLS UPGRADE TIMEOUT

if the same value is used for both, the 1st will be always triggered first in test ...

Reply:
marking as resolved because we no longer have `upgrade_timeout` in PR at all :-)